docs/home/contribute/style-guide.md

```diff
@@ -144,24 +142,56 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+### Versioning Kubernetes examples
+
+Code examples and configuration examples that include version information should be consistent with the accompanying text. Identify the Kubernetes version in the **Before you begin** section.
+
+To specify the Kubernetes version for a task or tutorial page:
+
+- Include `min-kubernetes-server-version` in the front matter of the page.
+- In the **Before you begin** section, use `{{ "{% include tasks-tutorial-prereqs.md "}} %}`.
+
+If the example YAML is in a standalone file, find and review the topics that include it as a reference.
+Verify that any topics using the standalone YAML have the appropriate version information defined.
+If a stand-alone YAML file is not referenced from any topics, consider deleting it instead of updating it.
+
+For example, if you are writing a tutorial that is relevant to Kubernetes version 1.8, the front-matter of your markdown file should look something like:
+
+```yaml
+---
+title: <your tutorial title here>
+min-kubernetes-server-version: v1.8
+---
+```
+
+In code and configuration examples, do not include comments about alternative versions.
+Be careful to not include incorrect statements in your examples as comments, such as:
+
+```yaml
+apiVersion: v1 # earlier versions use...
+kind: Pod
+...
+```
 
-{% comment %}## Kubernetes.io word list
+## Kubernetes.io word list
 
 A list of Kubernetes-specific terms and words to be used consistently across the site.
 
 <table>
   <tr><th>Term</th><th>Usage</th></tr>
-  <tr><td>TBD</td><td>TBD</td></tr>
-</table>{% endcomment %}
+  <tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
+  <tr><td>Docker</td><td>Docker should always be capitalized.</td></tr>
```

Comment:
```
<tr><td>Docker</td><td>Always capitalize Docker.</td></tr>
```

Reply:
How about "Capitalize Docker"?. Skip the "Always".